src/improvements/tasks/OPCMBaseTask.sol

```diff
@@ -95,7 +95,7 @@ abstract contract OPCMBaseTask is MultisigTask {
     /// @notice prank the multisig
     /// overrides MultisigTask to prank with delegatecall flag set to true
     function _prankMultisig() internal override {
-        vm.startPrank(parentMultisig, true);
```

Comment:
This was causing our issue.

Per the [docs](https://book.getfoundry.sh/cheatcodes/start-prank):

> startPrank(address msgSender, bool delegateCall): if delegateCall value is true then sets msg.sender for all subsequent delegate calls until [stopPrank](https://book.getfoundry.sh/cheatcodes/stop-prank.html) is called.

Inside the `upgrade` function on OPCM when `_opChainConfigs[i].systemConfigProxy.l1CrossDomainMessenger()` was called, we were not able to successfully retrieve the `l1CrossDomainMessenger`.

I reproduced the issue here: https://gist.github.com/blmalone/4f327167b95be99410651659ffc45845